content/v2.0/get-started.md

```diff
@@ -88,6 +88,109 @@ References to the InfluxDB user interface (UI) or localhost:9999 refer to your
 {{< cloud-name >}} UI.
 {{% /cloud %}}
 
+### (Optional) Download and install the influx CLI
+
+You can write, query, and process data through the Cloud user interface (UI) or  `influx` CLI. To download and install the `influx` CLI, complete the following steps:
+
+1. Click the **macOS** or **Linux** button below, and then click the appropriate **influx CLI** button to download the CLI binary.
+2. Do one of the following:
+   - For **macOS**: [Unpackage the binary](#unpackage-the-influx-binary), and then [(optional) place the binary in your $PATH](#optional-place-the-binary-in-your-path)
+   - For **Linux**: [Place the binary in your $PATH](#place-the-binary-in-your-path)
+3. If you're downloading on macOS Catalina, complete the steps in [Run influx CLI on macOS Catalina](/v2.0/get-started/#run-influx-cli-on-macos-catalina).
+4. (Optional) To install `influx` shell completion scripts, see [`influx completion`](/v2.0/reference/cli/influx/completion/#install-completion-scripts).
```

Comment:
This structure feels weird to me. Should we just nest the OS-specific content under each OS tab? I don't know that they need to be told to click the button. I think that can safely be assumed.

Reply:
thanks, @sanderson, I get what you're saying about not needing to tell them to click the button. My gut instinct when we start with "complete the following steps" is to include  something upfront so folks don't have to scan down -either numbered steps to indicate the order something needs to be done in or a bulleted list to indicate no specific order is required. That said, this section is pretty short and the corresponding content doesn't have numbered steps, so I took these out.